src/sonic-config-engine/sonic-cfggen

```diff
@@ -1,4 +1,19 @@
 #!/usr/bin/env python
+"""sonic-cfggen
+
+A tool to read SONiC config data from one or more of the following sources:
+minigraph file, config DB, json file(s), yaml files(s), command line input,
+and write the data into DB, print as json, or render a jinja2 config template.
+
+Examples:
+    Render template with minigraph:        
+        sonic-cfggen -m -t /usr/share/template/bgpd.conf.j2
+    Dump config DB content into json file: 
+        sonic-cfggen -d --print-data > db_dump.json
+    Load content of json file into config DB:
+        sonic-cfggen -j db_dump.json --write-to-db
```

Comment:
Let's keep the current option setting for dev purposes. I'll later add a "config load" and "config save" command to CLI for users.

---
In reply to: [128328144](https://github.com/Azure/sonic-buildimage/pull/808#discussion_r128328144) [](ancestors = 128328144)